src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CCDElasticSearchRepository.java

```diff
@@ -75,6 +76,15 @@ public CCDElasticSearchRepository(CoreCaseDataApi coreCaseDataApi,
 
     }
 
+    @Override
+    public List<Claim> getClaimsReadyForTransfer(User user) {
+        Query readyForTransferQuery = new Query(QueryBuilders.boolQuery()
+            .must(QueryBuilders.termQuery("state", ClaimState.READY_FOR_TRANSFER.getValue().toLowerCase()))
+            .must(QueryBuilders.existsQuery("data.hearingCourt"))
```

Comment:
The draw order process will set the `hearingCourt` field to null. We should be checking for `hearingCourtName` and `hearingCourtAddress`